src/HID-APIs/DefaultKeyboardAPI.hpp

```diff
@@ -24,46 +24,56 @@ THE SOFTWARE.
 // Include guard
 #pragma once
 
+DefaultKeyboardAPI::DefaultKeyboardAPI()
+{
+	memset(&_keyReport, 0, sizeof(_keyReport));
```

Comment:
As far as I know this is not required, as c++ should init all variable members to zero. Right?